micrometer-observation/build.gradle

```
@@ -1,3 +1,7 @@
+plugins {
+    id 'io.freefair.aspectj.post-compile-weaving' version '8.6'
```

this might be replaced with a reference declared in libs to be consistent with `micrometer-core`


```suggestion
    alias(libs.plugins.aspectj)
```